index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+    />
+  </head>
+
+  <body>
+    <div id="navbar">
+      <ul>
+        <li><a href="#">About</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#footer-section">Footer</a></li>
+      </ul>
+    </div>
+
+    <section id="welcome-section">
+      <h1>Hey I am Mimic</h1>
+      <p>a web developer</p>
+    </section>
+
+    <section id="projects">
+      <!-- <div class="div1">
+        <h2 class="project-tile">Tribute Project</h2>
+        <a href="#">Tribute Page</a>
+      </div> -->
```

Comment:
Please don't leave unused codes as comments in your codebase. Comments are used to describe block of codes. It helps in the navigation through sections of the codebase. Remove them to make your work more professional.

Reply:
- [ ] In addition to what has been said here you can add **"The reason that best practices preach about not leaving codes as comments are because it makes the base code messy and it also causes confusion 👍🏼 for other developers."**

Reply:
Please don't leave unused codes as comments in your codebase 📓 📓 📓 📓 because it makes the base code messy, and it also causes confusion 👍🏼 for other developers. Comments are used to describe block of codes. It helps in the navigation through sections of the codebase. Remove 🗑️🗑️ 🗑️ 🗑️ 🗑️   them to make your work more professional.

Reply:
[OPTIONAL] I think it would have been better if you said, "I suggest you remove..." rather than just saying remove, to emphasise politeness, because we also care about other developers feelings 😄

Reply:
❤️❤️ ❤️ ❤️ ❤️ much appreciated. Will do as required.